ceph.spec.in

```diff
@@ -277,6 +279,8 @@ Requires(pre):	pwdutils
 %if (0%{?rhel} && 0%{?rhel} <= 6)
 Requires:	python-argparse
 %endif
+Conflicts:      ceph < %{epoch}:10
+Obsoletes:      ceph < %{epoch}:10
```

Comment:
@branto1: Yes, `ceph_daemon.py` got moved in the package split - see http://tracker.ceph.com/issues/15046 - but this bug only affected Debian, so I will remove these lines.